markdown/dbaas.md

```diff
@@ -252,15 +246,83 @@ Once we registered kubernetes cluster we can use it’s name to create DB Cluste
+|kubernetes_cluster_name                |Kubernetes cluster name                         |Required                                                             |
+|name                                   |PXC cluster name to create                      |Default: pxc + DB version + 5 chars random string                    |
+|cluster_size                           |Cluster size                                    |Default: 3                                                           |
+|image                                  |Docker image name                               |Default is the recommended version from the Percona's version service|
+|compute_resources.cpu_m                |CPU resources millis                            |Default: 1000                                                        |
+|mcompute_resources.memory_bytes        |Max memory size in bytes                        |Default: 2 Gb                                                        |
+|disk_size                              |Max disk size for the PXC instance              |Default: 25 Gb                                                       |
+|proxysql.image                         |Docker image for ProxySQL                       |Default: empty. (Use operator's default)                             |
+|proxysql.compute_resources.cpu_m       |CPU resources millis                            |Default: 1000                                                        |
+|proxysql.compute_resources.memory_bytes|Max memory size in bytes                        |Default 2 Gb                                                         |
+|proxysql.disk_size                     |Max disk size for ProxySQL                      |Default: empty, use operator's default                               |
+|haproxyimage                           |Docker image for HA Proxy                       |Default: empty, use operator's default                               |
+|haproxy.compute_resources.cpu_m        |CPU resources millis                            |Default: 1000                                                        |
+|haproxy.compute_resources.memory_bytes |Max memory size in bytes                        |Default: 2 Gb                                                        |
+|expose                                 |Make it available outside the Kubernetes cluster|Default: false                                                       |
+
+**Note:** 
+Only one of ProxySQL or HAProxy should be specified in the request.
+
+#### Minimum request example
+
+Since the API has the defaults mentioned above, the HTTP request can have the kubernetes cluster name as the only parameter.
+
+Example:
+
+```bash
+curl -X POST "http://localhost/v1/management/DBaaS/PXCCluster/Create" \
+    -H "accept: application/json" \
+    -H "authorization: Basic YWRtaW46YWRtaW4=" \
+    -H "Content-Type: application/json" \
+    -d '{ "kubernetes_cluster_name": "my_cluster", "name": "my-pxc-cluster", "expose": true}'
```

Comment:
"..name as the only parameter.."
but in example - we have three parameters. ) 

Let's keep this the simples one?